config/example-config.exs

```diff
@@ -3,6 +3,10 @@ use Mix.Config
 config :hermetic, Hermetic,
   cowboy_options: [port: 59468]
 
+config :hermetic, Hermetic.Slash.Deploy,
+  default_ref: "master",
+  default_env: "production"
```

Comment:
While I'm aware that this is configurable, I'd prefer the default to be staging, so that a deployment to production needs to be explicitly specified.

Reply:
I chose `production` because this is what the github api defaults to, so I think it makes more sense because that is what people expect a bot to do that just wraps the github deployments api.